buildenv/jenkins/JenkinsfileBase

```diff
@@ -182,6 +182,7 @@ def post() {
 				if (env.BUILD_LIST == 'jck') {
 					sh 'tar -zcf jck-test-results.tar.gz $WORKSPACE/openjdk-tests/TestConfig/test_output_*'
 					archiveArtifacts artifacts: '**/jck-test-results.tar.gz', fingerprint: true, allowEmptyArchive: true
+					cleanWs()
```

Comment:
We have cleanWS() at the end of build jobs no matter what tests is running. Why need extra cleanWs()?